dataprofiler/tests/profilers/test_json_encoder.py

```diff
@@ -89,3 +89,51 @@ def test_encode_categorical_column_profiler_after_update(self):
         )
 
         self.assertEqual(serialized, expected)
+
```

Comment:
Thinking long term, would it be better to put the profile specific tests with each profiler's tests instead?

Reply:
I like having the tests match the file structure so yeah I second this for sure!